FWCore/PythonFramework/test/runPythonTests.sh

```diff
@@ -12,4 +12,10 @@ do
 done
 
 echo "running test_producer.py"
-python ${LOCAL_TEST_DIR}/test_producer.py || die "test_producer.py failed"
+{ 
+python3 -c "from FWCore.PythonFramework.CmsRun import CmsRun" 2>/dev/null && python3 ${LOCAL_TEST_DIR}/test_producer.py || die "test_producer.py failed"
+}|| 
+{ 
+python -c "from FWCore.PythonFramework.CmsRun import CmsRun" 2>/dev/null && python2 ${LOCAL_TEST_DIR}/test_producer.py || die "test_producer.py failed"
+} 
```

Comment:
I'm a little bit confused why this is needed (likely because of being too ignorant for some details). Is `FWCore.PythonFramework` going to use python3 in the nominal IB that uses python2 by default, or something?

Reply:
Nevermind, https://github.com/cms-sw/cmsdist/pull/7019#issuecomment-862119576 gave the context.